llvm/lib/Target/Lanai/LanaiRegisterInfo.h

```diff
@@ -43,8 +43,6 @@ struct LanaiRegisterInfo : public LanaiGenRegisterInfo {
   Register getFrameRegister(const MachineFunction &MF) const override;
   Register getBaseRegister() const;
   bool hasBasePointer(const MachineFunction &MF) const;
-
-  int getDwarfRegNum(unsigned RegNum, bool IsEH) const;
```

Comment:
why did you remove this?

Reply:
It caused errors when I switched `getDwarfRegNum` to be virtual, and there didn't seem to be a reason for it to be there.  No tests failed from removing it.  I believe it is vestigial, but I'm open to someone telling or showing me why it should be there.

Reply:
No worries. What you say makes sense.